http_client.rst

```diff
@@ -148,7 +148,8 @@ method to retrieve a new instance of the client with new default options::
 
 .. versionadded:: 5.3
 
-    The ``withOptions()`` method was introduced in Symfony 5.3.
+    The :method:`Symfony\\Contracts\\HttpClient\\HttpClientInterface::withOptions`
```

Comment:
I would not do this change. The method is link 1 sentence before this one, it does not add much (and versionadded directives never explain new information, they only document a version)

Reply:
Reverted 👍